index.html

```diff
@@ -0,0 +1,464 @@
+            <div class="issues">
+                <p>CMS <sup>20</sup></p>
+                <p>JavaScript <sup>290</sup></p>
+                <p>Framework <sup>64</sup></p>
+            </div>
+            <div>
+                <p class="arrow">→</p>
+            </div>
+        </article>
+    </article>
+    <article class="article-grid">
+        <article class="grid-elem">
+            <div class="article-header">
+                <p>6 days ago</p>
+                <a class="grid-elem-title" href="#">A CSS Grid Framework For Shopify Collection Pages</a>
+                <p>by <a class="grid-elem-author" href="#">Liam Griffin</a></p>
+            </div>
+            <p>In this artticle, we'll be looking at how to set up a grid layout products
+                on your collection pages, and how to use Shopify's section settong to create
+                customizable options in the online store editor.
+            </p>
+            <button class="arrow-botton">→</button>
+        </article>
+        <article class="grid-elem">
+            <div class="article-header">
+                <p>6 days ago</p>
+                <a class="grid-elem-title" href="#">A CSS Grid Framework For Shopify Collection Pages</a>
+                <p>by <a class="grid-elem-author" href="#">Liam Griffin</a></p>
+            </div>
+            <p>In this artticle, we'll be looking at how to set up a grid layout products
+                on your collection pages, and how to use Shopify's section settong to create
+                customizable options in the online store editor.
+            </p>
+            <button class="arrow-botton">→</button>
+        </article>
+        <article class="grid-elem">
+            <div class="article-header">
+                <p>6 days ago</p>
+                <a class="grid-elem-title" href="#">A CSS Grid Framework For Shopify Collection Pages</a>
+                <p>by <a class="grid-elem-author" href="#">Liam Griffin</a></p>
+            </div>
+            <p>In this artticle, we'll be looking at how to set up a grid layout products
+                on your collection pages, and how to use Shopify's section settong to create
+                customizable options in the online store editor.
+            </p>
+            <button class="arrow-botton">→</button>
+        </article>
+    </article>
+    <article class="advertise">
+      <article class="shopify">
+        <p class="ofs">Our friends and supporters.</p>
+        <div class="partner">
+          <div class="partner-title">
+            <p><span class="shop">shopify </span> partners </p>
+          </div>
+            <div class="partner-des">
+              <p>Earn recurring revenue designing and</p>
+              <p>developing eCommerce stores</p>
+
+            </div>
+
+            <a href="#" class="sign">SIGN UP TODAY</a>
+
+        </div>
```

Comment:
- Watch out for the unnecessary white space. This should be like a newspaper the white space should improve readability not break it.